Engine/Controller/Regulators/velocity_regulators.py

```diff
@@ -21,8 +21,8 @@
 }
 
 if Config()['COACH']['type'] == 'sim':
-    settings['orientation_pid_settings'] = {'kp': 2, 'ki': 0.3, 'kd': 0}
-    settings['v_d'] = 15
+    settings['orientation_pid_settings'] = {'kp': 10, 'ki': 3, 'kd': 0.3}
```

Comment:
move to `VelocityRegulator.__init__`

Reply:
Aussi inversé les constantes pour la simultion et pour la non simulation.

Reply:
Aussi au lieu de `Config()['COACH']['type'] == 'sim'` utilise `Config().is_simulation()`